packages/tools/guiEditor/src/controlTypes.ts

```diff
@@ -91,3 +91,80 @@ export const ControlTypes = [
         icon: colorPickerIcon,
     },
 ];
+
+export const ControlTypesFHalf = [
+    {
+        className: "Rectangle",
+        icon: rectangleIcon,
+    },
+    {
+        className: "Ellipse",
+        icon: ellipseIcon,
+    },
+    {
+        className: "StackPanel",
+        icon: stackPanelIcon,
+    },
+    {
+        className: "Grid",
+        icon: gridIcon,
+    },
+    {
+        className: "ScrollViewer",
+        icon: scrollbarIcon,
+    },
+    {
+        className: "Line",
+        icon: lineIcon,
+    },
+    {
+        className: "TextBlock",
+        icon: textBoxIcon,
+    },
+    {
+        className: "InputText",
+        icon: inputFieldIcon,
+    },
+    {
+        className: "InputPassword",
+        icon: passwordFieldIcon,
+    },
+];
+export const ControlTypesSHalf = [
+    {
+        className: "Image",
+        icon: imageIcon,
+    },
+    {
+        className: "DisplayGrid",
+        icon: displaygridIcon,
+    },
+    {
+        className: "Button",
+        icon: buttonIcon,
+    },
+    {
+        className: "Checkbox",
+        icon: checkboxIcon,
+    },
+    {
+        className: "RadioButton",
+        icon: radioButtonIcon,
+    },
+    {
+        className: "Slider",
+        icon: sliderIcon,
+    },
+    {
+        className: "ImageBasedSlider",
+        icon: imageBasedSliderIcon,
+    },
+    {
+        className: "VirtualKeyboard",
+        icon: keyboardIcon,
+    },
+    {
+        className: "ColorPicker",
+        icon: colorPickerIcon,
+    },
+];

```

Comment:
these parameters can be added to ControTypes instead of creating new arrays with almost the same elements. You could add new keys to ControlTypes (for example `halfIcon:...` that will hold the icon for the extra parameters that were added).